README.md

```diff
@@ -2175,7 +2175,7 @@ decide what you should do.
 
 - **Exercise and hold**: You can write the company a check and pay any taxes on the spread.
   You are then a stockholder, with a stock certificate that may have value in the future.
-  As [discussed](#equity-compensation-details), you may exercise:
+  As [discussed](#how-equity-is-granted), you may exercise:
```

Comment:
How Equity Is Granted contains all info on vesting and exercising (inc. early exercise), but lmk if you think there's a more specific section this could link to.